### ISSUE TITLE
[Bug] Can't use SukiUI.Dock 6.0.0-beta7 becasuse of System.MissingMethodException

### ISSUE THREAD

**Issue by orangebobolink** [kikipoulet/SukiUI#353]
### Check the following items

- [X] I have looked up relevant Issue

### Description of the issue

When i try to use <StyleInclude Source="avares://SukiUI.Dock/Index.axaml" /> this provoke 

Unhandled exception. System.MissingMethodException: Method not found: 'System.Collections.Generic.IReadOnlyList`1<System.Object> Avalonia.Markup.Xaml.XamlIl.Runtime.IAvaloniaXamlIlEagerParentStackProvider.get_DirectParents()'.
   at CompiledAvaloniaXaml.!AvaloniaResources.Populate:/Index.axaml(IServiceProvider, Styles)
   at CompiledAvaloniaXaml.!AvaloniaResources.Build:/Index.axaml(IServiceProvider)
   at Ias.AvaloniaUi.App.!XamlIlPopulate(IServiceProvider, App) in D:\Projects\ias\src\Ias.AvaloniaUi\App.axaml:line 16
   at Ias.AvaloniaUi.App.!XamlIlPopulateTrampoline(App)
   at Ias.AvaloniaUi.App.Initialize() in D:\Projects\ias\src\Ias.AvaloniaUi\App.axaml.cs:line 13
   at Avalonia.AppBuilder.SetupUnsafe()
   at Avalonia.AppBuilder.Setup()
   at Avalonia.AppBuilder.SetupWithLifetime(IApplicationLifetime lifetime)
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime(AppBuilder builder, String[] args, Action`1 lifetimeBuilder)
   at Ias.AvaloniaUi.Program.Main(String[] args) in D:\Projects\ias\src\Ias.AvaloniaUi\Program.cs:line 13
   
 Without <StyleInclude Source="avares://SukiUI.Dock/Index.axaml" /> Avolonia can run.
 I tried to upgrade/downgrade nugets but it didn't help with this issue
 I use .net 8
 
![image](https://github.com/user-attachments/assets/1b15039a-2956-4584-a904-f97d8425eb2a)

### Package Version

6.0.0-rc

### Environment

Windows 10 Pro 22H2

### Expected Behavior

Available use SukiUI.Dock with <StyleInclude Source="avares://SukiUI.Dock/Index.axaml" />

### Reproduction

1. Create Avalonia project with 11.2.2 version and .net 8.
2. Download SukiUI 6.0.0-beta8 and SukiUI.Dock 6.0.0-beta7
3. Write code in App.axaml
```csharp
<Application xmlns="https://github.com/avaloniaui"
             xmlns:sukiUi="clr-namespace:SukiUI;assembly=SukiUI"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             x:Class="Ias.AvaloniaUi.App"
             xmlns:local="using:Ias.AvaloniaUi"
             RequestedThemeVariant="Default">
             <!-- "Default" ThemeVariant follows system theme variant. "Dark" or "Light" are other available options. -->
     

    <Application.DataTemplates>
        <local:ViewLocator/>
    </Application.DataTemplates>
  
    <Application.Styles>
        <sukiUi:SukiTheme ThemeColor="Blue"  />
        <StyleInclude Source="avares://SukiUI.Dock/Index.axaml" />
    </Application.Styles>
</Application>
```
4. Run app

**Comment by kikipoulet** [comment]
I have published the 6.0.0-rc nuget package for SukiUI.Dock only right now, you can try

**Comment by orangebobolink** [comment]
That's work. Thank you)
